functions/src/firestore/updateBookInAlgolia/index.js

```diff
@@ -10,8 +10,11 @@ module.exports = functions.firestore.document('books/{uid}').onUpdate((change, c
         releaseDate,
         isbn,
         language,
+        starts,
     } = change.after.data();
 
+    const ratingCount = Object.values(starts).reduce((pv, cv) => Number(pv) + Number(cv), 0);
```

Comment:
let's put this in a function getRatingCount (inside functions/src/helpers/getRatingCount.js)